### PR TITLE
fetch status from crowdin

### PR DESCRIPTION
**Pull request by maddhruv**

### CODE REVIEW

File: script/status.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const fs = require('fs')
+const path = require('path')
+const request = require('request')
+
+const projectKey
```
Comment (TiagoDanin) [inline script/status.js]: Use the `dotenv-safe` to get the key

Comment (zeke) [reply script/status.js]: Yeah I agree. Here's how to set up `dotenv-safe`

```sh
npm i -D dotenv-safe
echo "CROWDIN_KEY=" > .env.example
echo "CROWDIN_KEY=s3cr3tz" > .env
echo ".env" >> .gitignore
```

Then in status.js:

```js
require('dotenv-safe').load()
const key = process.env.CROWDIN_KEY
```

Comment (maddhruv) [reply script/status.js]: done :zap:

File: package.json [hunk 2]
```diff
@@ -35,5 +35,8 @@
     "env": [
       "jest"
     ]
+  },
+  "dependencies": {
```
Comment (zeke) [inline package.json]: This should be in `devDependencies`. Ideally when we ship #92, the published module will not have any runtime dependencies.

**Comment by zeke** [comment]
Commented on this last night, but somehow the comment got lost.

Can you add a `.env.example` file with `CROWDIN_KEY=` and a `.gitignore` file with `.env`?

**Comment by maddhruv** [comment]
> Can you add a .env.example file with CROWDIN_KEY= and a .gitignore file with .env

@zeke we already have that

**Comment by zeke** [comment]
> we already have that

Ah! Sorry. Curious: did you delete one of my comments?

**Comment by maddhruv** [comment]
@zeke nope!

**Comment by maddhruv** [comment]
Will ignore the yarn.lock

**Comment by maddhruv** [comment]
@zeke can we do this?

**Comment by maddhruv** [comment]
thanks BOSS

**Comment by zeke** [comment]
No thank you! Also I'm not the boss. We're all in this together. 💛

**Pull request merged**